chore: ungate multi-column keys from the feature flag (#6752)
confluentinc/ksql: docs/developer-guide/serialization.md

For example, given a SQL statement such as:

```sql
CREATE STREAM x (ORGID BIGINT KEY, ID BIGINT KEY, NAME STRING, AGE INT) WITH (FORMAT='DELIMITED', ...);
```

> ksqlDB splits a key of `120, 21` and a value of `bob, 49` into the four fields (two keys and two values) 

Reviewer: Is it valid to have spaces here? I thought the default delimiter was a comma (with no space)?

Author: I think you are right!

### (de)serialization of single keys

> ksqlDB assumes that any single key is unwrapped, i.e. not contained within an outer

Reviewer 2: ```suggestion
ksqlDB assumes that any single key is unwrapped, which mean that it's not contained in an outer
```

record or object. Conversely, ksqlDB assumes that any key with multiple columns

> (e.g. `CREATE STREAM K1 INT KEY, K2 INT KEY, C1 INT`) is wrapped, i.e is a record

Reviewer 2: ```suggestion
(for example, `CREATE STREAM K1 INT KEY, K2 INT KEY, C1 INT`) _is_ wrapped, which means that it is a record
```

with each column as a field within the key. 

> To declare a single column key that is wrapped, you may specify a `STRUCT` type

Reviewer 2: ```suggestion
To declare a single-column key that's wrapped, specify a `STRUCT` type
```

> with a single column (e.g. `K STRUCT<F1 INT> KEY`). See the next two sections 

Reviewer 2: ```suggestion
with a single column. for example, `K STRUCT<F1 INT> KEY`. See the next two sections 
```
